app/components/pages/UserProfile.jsx

```diff
@@ -152,46 +138,46 @@ export default class UserProfile extends React.Component {
         if( section === 'transfers' ) {
             walletClass = 'active'
             tab_content = <div>
-                <UserWallet
-                          account={accountImm}
-                          showTransfer={this.props.showTransfer}
-                          current_user={current_user}
-                          withdrawVesting={this.props.withdrawVesting} />
+                <UserWallet global={this.props.global}
```

Comment:
Is using the global here necessary? @svk31 has been cutting down on passing around the whole state for performance.